src/internal-links/handler.js

```diff
@@ -78,7 +78,7 @@ export async function internalLinksAuditRunner(auditUrl, context) {
     granularity: 'hourly',
   };
 
-  log.info(`${auditType}: Options for RUM call: `, JSON.stringify(options));
+  log.info(`broken-internal-links audit: ${auditType}: Options for RUM call: `, JSON.stringify(options));
```

Comment:
why this change?

Reply:
Because of lot of similar logs, I need to filter them sometimes. This is for filtering and adding some clarity.